ProcessLib/BoundaryCondition/NeumannBoundaryConditionLocalAssembler.h

```diff
@@ -26,11 +28,11 @@ class UniformNeumannBoundaryConditionLocalAssembler final
 public:
     /// The neumann_bc_value factor is directly integrated into the local
     /// element matrix.
-    UniformNeumannBoundaryConditionLocalAssembler(
+    NeumannBoundaryConditionLocalAssembler(
         MeshLib::Element const& e,
         std::size_t const local_matrix_size,
         unsigned const integration_order,
-        double neumann_bc_value)
+        Parameter<double> const& neumann_bc_value)
```

Comment:
renamed to `neumann_bc_parameter`. Plain `parameter` would be rather unspecific, I think.